cnames_active.js

```diff
@@ -3490,6 +3490,7 @@ var cnames_active = {
   "with-immutable-props-to-js": "tophat.github.io/with-immutable-props-to-js",
   "within": "eric-brechemier.github.io/within", // noCF? (don´t add this in a new PR)
   "wiz": "karimsa.github.io/wiz", // noCF
+  "wizzl": "wizzldev.github.io/wizzl.js"
```

Comment:
```suggestion
  "wizzl": "wizzldev.github.io/wizzl.js",
```
As the CI error notes, missing a comma

Reply:
Sorry about that, I fixed it.